python/pyspark/sql/dataframe.py

```diff
@@ -612,9 +614,10 @@ def repartition(self, numPartitions, *cols):
         Returns a new :class:`DataFrame` partitioned by the given partitioning expressions. The
         resulting DataFrame is hash partitioned.
 
-        ``numPartitions`` can be an int to specify the target number of partitions or a Column.
-        If it is a Column, it will be used as the first partitioning column. If not specified,
-        the default number of partitions is used.
+        :param numPartitions:
+            can be an int to specify the target number of partitions or a Column.
```

Comment:
I think it's fine to:

```
        :param numPartitions: can be an int to specify the target number of partitions or a Column.
            If it is a Column, it will be used as the first partitioning column. If not specified,
```

Reply:
Cool, was maintaining consistency with other `:param` on page.